wordpress-mobile/WordPress-iOS
# Update Aztec pod to beta 20

Updates Aztec to beta 20. This brings the following updates:

- Gutenberg blocks with multiple HTML elements are know correctly output.
- Self closing Gutenberg block are read and output correctly, presentation still needs to be improved
- Updates on the processor for shortcodes that keeps all the attributes from the original shortcodes

To test:
 - Start the app
 - Open posts with regular Calypso content and check that it shows ok
 - Open posts with Gutenberg content and see that they work correctly
 - Create a new post on the device with media (images and videos) and see if it works correctly
